### PR TITLE
Populated Course Details Page

### LINKED ISSUE #24: Priority: Populate Placeholder Page with Table for CourseDetailsPage
- Modify placeHolder so that it now shows a table with course details.
- The table will show detailed implemented in issue #14 and Issue #23 
- Relies on Issue #18

### PR DESCRIPTION
**Pull request by ivan512az**
In this PR, we are populating the CourseDetailsPage with the CourseDetailsTable that contains information from the course. 
Following the path **/coursedetails/yyyyq/enrollCd**, the CourseDetailsPage shows the information available from the course.
Here is a picture of the page populated with information:
Screenshot 2023-11-30 at 10 47 55 AM

Here is the link to dokku: https://proj-courses-rosillo-m-dev.dokku-11.cs.ucsb.edu
Closes #24

### CODE REVIEW

**Review by ThatsBadJuju — APPROVED** [review]
LGTM

File: frontend/src/main/pages/CourseDetails/CourseDetailsPage.js [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
 export default function CourseDetailsPage() {
-  // Stryker disable all : placeholder for future implementation
+  let { yyyyq, enrollCd } = useParams();
+
+  const {
+    data: courseDetail,
+    _error,
+    _status,
+  } = useBackend(
+    // Stryker disable all : hard to test for query caching
```
Comment (pconrad) [inline frontend/src/main/pages/CourseDetails/CourseDetailsPage.js:14]: This should be matched with a `// Stryker restore all` as soon as the part you are not testing for is done.   Where is the matching line?

Comment (pconrad) [reply frontend/src/main/pages/CourseDetails/CourseDetailsPage.js:14]: Or change it to the `next-line` version

**Review by pconrad — CHANGES_REQUESTED** [review]
Code tested on Dokku; works fine.  One issue: check the // Stryker exceptions

File: frontend/src/main/pages/CourseDetails/CourseDetailsPage.js (hunk 1 above)
Comment (ThatsBadJuju) [reply frontend/src/main/pages/CourseDetails/CourseDetailsPage.js]: Changed to next-line version!

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**